ansible/idr-elasticsearch.yml

```diff
@@ -0,0 +1,164 @@
+# configure and run Elasticsearch for the searchengine
+
+- hosts: "{{ idr_environment | default('idr') }}-database-hosts"
```

Comment:
Loading the host variables from the database VM was used in the searchengine playbook for setting the `database_server_url` variable.
Is it correct that it is not required at all for setting up Elasticsearch? In this case, this line can simply be removed

Reply:
I have removed it from this playbook.